archaius2-core/src/main/java/com/netflix/archaius/config/PollingDynamicConfig.java

```diff
@@ -75,15 +73,15 @@ private void update() {
                 PollingResponse response = reader.call();
                 if (response.hasData()) {
                     current = response.getToAdd();
-                    notifyConfigUpdated();
+                    notifyConfigUpdated(this);
                 }
             }
             catch (Exception e) {
                 try {
-                    notifyError(e);
+                    notifyError(e, this);
                 }
                 catch (Exception e2) {
-                    LOG.warn("Failed to notify error observer for config " + getName());
+                    LOG.warn("Failed to notify error observer");
```

Comment:
Shouldn't this pass the exception to the logger so the user will know what the failure was?